lib/oli/delivery/attempts/core.ex

```diff
@@ -868,6 +881,18 @@ defmodule Oli.Delivery.Attempts.Core do
     |> Repo.update()
   end
 
+  @doc """
+  Updates all resource accesses for a given section.
+  """
+
+  def update_resource_accesses_by_section(section_id, new_section_id) do
```

Comment:
I don't think this is being used, and it would be dangerous to use it as-is since it is not user specific. We should remove it.